app/text_buffer.py

```diff
@@ -607,8 +610,12 @@ def find(self, searchFor, direction=0):
       self.doSelectionMode(app.selectable.kSelectionNone)
       return
     # The saved re is also used for highlighting.
-    self.findRe = re.compile('()'+searchFor)
-    self.findBackRe = re.compile('(.*)'+searchFor)
+    if self.ignoreCaseFind:
```

Comment:
I've moved the BackingTextBuffer into ci_edit/app/actions.py, are ya cool with making the changes there?